app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, foreign_key: :post_id
+  has_many :comments, foreign_key: :post_id
+
+  after_save :update_post_counter
+
+  def five_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_post_counter
+    author.update(posts_counter: author.posts.all.length)
+  end
```

Comment:
- [ ] [OPTIONAL] Good approach to solving this problem. I'd like to suggest a shorter and even more readable way to do this by using the `increment` method as shown below 👇 
``` ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```

Reply:
Thank you namesake @codepantha 
I have implemented all optional changes.

Reply:
Yeah, you stole my name 🤣. And you're welcome 👍